fiftyone/server/view.py

```diff
@@ -84,7 +84,8 @@ def get_view(
     """Gets the view defined by the given request parameters.
 
     Args:
-        dataset_name: the dataset name
+        dataset: the dataset name or :class:`fiftyone.core.dataset.Dataset`
+        instance
```

Comment:
nit: indent second, third etc lines of docstrings as done elsewhere. This is required for Sphinx docstrings to render properly